app/lib/shared/dartdoc_memcache.dart

```diff
@@ -24,7 +24,6 @@ DartdocMemcache get dartdocMemcache =>
 
 class DartdocMemcache {
   final SimpleMemcache _entry;
-  final SimpleMemcache _fileInfo;
```

Comment:
There is a benefit of keeping related caches close together, otherwise sooner or later we'll ask: what kind of caches are we using for X? and we'll need to hunt them down with more effort.

If we want to change this specific pattern of grouping, let's do so only with a clear idea where we are heading (consistently with other similar cache groups). If the goal is to replace `SimpleCache` with `cache`, let's do it here, and not move it yet.

Reply:
Good point, I'll play around with this.